project/app/views.py

```diff
@@ -11,31 +11,32 @@
 @api_view(['GET'])
 def apiOverview(request):
     api_urls = {
-        'List': '/task-list/',
-        'Detail View': '/task-detail/<str:pk>/',
-        'Create': '/task-create/',
-        'Update': '/task-update/<str:pk>/',
-        'Delete': '/task-delete/<str:pk>/',
+        'sign-up': '/sign-up/',
+        'login': '/login/',
     }
     return Response(api_urls)
 
 
-@api_view(['GET'])
-def taskList(request):
-    tasks = Task.objects.all()
-    serializer = TaskSerializer(tasks, many=True)
-    return Response(serializer.data)
-
-
-@api_view(['GET'])
-def viewone(request, pk):
-    tasks = Task.objects.get(id=pk)
-    serializer = TaskSerializer(tasks,many=False)
+@api_view(['POST'])
+def signUp(request):
+    serializer = SignUpSerializer(data=request.data)
+    if serializer.is_valid():
+        serializer.save()
```

Comment:
What happen is case of false if condition? Put Invalid data in API's request body and test once.